src/graph/topo.cc

```diff
@@ -1360,7 +1364,7 @@ static ncclResult_t ncclTopoPopulateNics(ncclXml* xml, int startIndex, int endIn
 
 // Calls to network plugin APIs should be protected. This function should be called inside a per-process lock.
 ncclResult_t ncclTopoProcessNet(ncclXml* xml, int coll, const char* dumpXmlFile, ncclTopoNetState* state, ncclResult_t (*getProperties)(int, ncclNetProperties_t*), ncclResult_t (*makeVDevice)(int*, ncclNetVDeviceProps_t*), ncclResult_t (*devices)(int*), const char* netName, bool dmaBufSupport) {
-  int usePhysicalDevices = (dumpXmlFile || makeVDevice == NULL);
+  int usePhysicalDevices = (makeVDevice == NULL); /*(dumpXmlFile || makeVDevice == NULL);*/
```

Comment:
```suggestion
  int usePhysicalDevices = (dumpXmlFile || makeVDevice == NULL);
```
We should maintain the old xml dump functionality.